src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineBus.java

```diff
@@ -75,4 +76,9 @@ public void updateState(LfNetworkStateUpdateParameters parameters) {
 
         super.updateState(parameters);
     }
+
+    @Override
+    public ViolationLocation getViolationLocation() {
+        return null;
+    }
```

Comment:
to prevent from implementing should it be better to have a default implementation in the interface lfBus ?

Reply:
ok